test_programs/execution_success/sha256_var_witness_const_regression/src/main.nr

```diff
@@ -1,9 +0,0 @@
-fn main(input: [u8; 2], toggle: bool) {
-    let size: Field = 1 + toggle as Field;
-    assert(!toggle);
-
-    let variable_sha = std::sha256::sha256_var(input, size as u64);
```

Comment:
I removed this test as this exact same regression is done under `execution_success/sha256`